src/Core/Auth/Models/Data/SsoConfigurationData.cs

```diff
@@ -22,7 +22,25 @@ public string Serialize()
 
     public SsoType ConfigType { get; set; }
 
-    public bool KeyConnectorEnabled { get; set; }
+    public MemberDecryptionType MemberDecryptionType { get; set; }
+
+    /// <summary>
+    /// Legacy property to determine if KeyConnector was enabled.
+    /// Kept for backwards compatibility with old configs that will not have
+    /// the new MemberDecryptionType when deserialized from the database.
```

Comment:
Biggest nit ever but could we make it `the new <see cref="MemberDecryptionType" /> when`